public/pages/mp-page.html

```diff
@@ -1,25 +1,45 @@
 <!DOCTYPE html>
 <script src="https://unpkg.com/react@18/umd/react.development.js"></script>
 <script src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
+<script src="https://unpkg.com/i18next@latest/dist/umd/i18next.js"></script>
+<script src="https://unpkg.com/react-i18next@latest/dist/umd/react-i18next.js"></script>
+<script src="https://unpkg.com/i18next-browser-languagedetector@latest/dist/umd/i18nextBrowserLanguageDetector.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/i18next-http-backend@1.3.1/i18nextHttpBackend.min.js"></script>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>MP Details</title>
+    <title>Is My MP a Landlord? | MP Detail</title>
```

Comment:
Generally, I think it’s better to front-load titles. That way truncation doesn’t obscure the most significant information.

```suggestion
    <title>MP Detail | Is My MP a Landlord?</title>
```